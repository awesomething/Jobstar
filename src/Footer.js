import React from 'react';
import './Footer.css'

function Footer() {
    return (
        <section>
        <footer className="footer">
          <div className="footer">
            <div>
              
              <a href="https://www.linkedin.com/in/henryebomah/" 
              target="_blank" rel="noopener noreferrer">LinkedIn<i className="fab fa-linkedin"></i></a>
            </div>
            <div className="copyright-text">
              <p>
                Copyright &copy; 2021 | JobLobster
              <br />
                All Rights Reserved
            </p>
            
            </div>
          </div>
        </footer>
        </section>
    )
}

export default Footer