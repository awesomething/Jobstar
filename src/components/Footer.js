import React from 'react';
import './Footer.css'

function Footer() {
    return (
        
        <footer className="footer">
          <div className="footer">
            
          
              
              <a href="https://www.linkedin.com/in/henryebomah/" 
              target="_blank" rel="noopener noreferrer">LinkedIn<i className="fab fa-linkedin"></i></a>
            </div>
            <div className="copyright-text">
              <p>
                Copyright &copy; 2021 | Jobstar
              <br />
                All Rights Reserved
            </p>
            
            </div>
          
        </footer>
        
    )
}

export default Footer