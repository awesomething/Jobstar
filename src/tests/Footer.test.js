import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Footer from '../components/Footer'



describe(`Footer component`, () => {
  const props = {
    id: 'a',
    name: 'test-class-name',
    
  }

  it('renders a .Footer by default', () => {
    const wrapper = shallow(<Footer />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })

  it('renders the Footer given props', () => {
    const wrapper = shallow(<Footer {...props} />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
