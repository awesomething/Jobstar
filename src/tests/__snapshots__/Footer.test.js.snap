// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders a .Footer by default 1`] = `
<footer
  className="footer"
>
  <div
    className="footer"
  >
    <a
      href="https://github.com/awesomething"
      rel="noopener noreferrer"
      target="_blank"
    >
      GitHub
      <i
        className="fab fa-github"
      />
    </a>
    <br />
    <a
      href="https://www.linkedin.com/in/henryebomah/"
      rel="noopener noreferrer"
      target="_blank"
    >
      LinkedIn
      <i
        className="fab fa-linkedin"
      />
    </a>
  </div>
  <div
    className="copyright-text"
  >
    <p>
      Copyright © 2021 | Jobstar
      <br />
      All Rights Reserved
    </p>
  </div>
</footer>
`;

exports[`Footer component renders the Footer given props 1`] = `
<footer
  className="footer"
>
  <div
    className="footer"
  >
    <a
      href="https://github.com/awesomething"
      rel="noopener noreferrer"
      target="_blank"
    >
      GitHub
      <i
        className="fab fa-github"
      />
    </a>
    <br />
    <a
      href="https://www.linkedin.com/in/henryebomah/"
      rel="noopener noreferrer"
      target="_blank"
    >
      LinkedIn
      <i
        className="fab fa-linkedin"
      />
    </a>
  </div>
  <div
    className="copyright-text"
  >
    <p>
      Copyright © 2021 | Jobstar
      <br />
      All Rights Reserved
    </p>
  </div>
</footer>
`;
